connectors/src/api/connector_config.ts

```diff
@@ -9,7 +9,7 @@ import {
   SET_CONNECTOR_CONFIG_BY_TYPE,
 } from "@connectors/connectors";
 import { apiError, withLogging } from "@connectors/logger/withlogging";
-import { ConnectorModel } from "@connectors/resources/storage/models/connector_model";
+import { ConnectorResource } from "@connectors/resources/connector_res";
```

Comment:
not a big fan of `connector_res`. `res` is being used heavily in the codebase as a shorthand for `result`

Reply:
Done!